backend/dal/Helpers/Extensions/ProjectExtensions.cs

```diff
@@ -400,7 +400,7 @@ public static void Merge(this Entity.Project originalProject, Entity.Project upd
             var updatedById = originalProject.UpdatedById;
             context.Entry(originalProject).CurrentValues.SetValues(updatedProject);
             originalProject.Agency = agency; // Don't want to allow agency to change through this method.
-            originalProject.CreatedById = updatedById; // Don't want these updated externally.
```

Comment:
the updatedById used to be createdById, I think it was a mistake?
This seems to only occur on a project approval, I couldn't get it to happen when just updating a project.

Reply:
You're probably correct.